packages/dds/ordered-collection/src/consensusOrderedCollectionFactory.ts

```diff
@@ -14,6 +14,7 @@ import { pkgVersion } from "./packageVersion.js";
 
 /**
  * The factory that defines the consensus queue
+ * @internal
```

Comment:
Why the mismatch between the class and the functions below?  Seems off.  I thought classes/members had to all have the same tags.

Reply:
Just an oversight, I'll update it

Reply:
Updated to match what was in main/rc2 branches before: class tagged `@internal`, no tags on the functions